tsconfig.json

```diff
@@ -1,16 +1,20 @@
 {
   "$schema": "https://json.schemastore.org/tsconfig",
-  "extends": "@tsconfig/next/tsconfig.json",
   "compilerOptions": {
+    "lib": ["dom", "dom.iterable", "esnext"],
```

Comment:
Do we want this / need this anymore? I think `dom` and `dom.iterable` don't work anymore since we don`t use next react.

```suggestion
```

Reply:
they still work, they're provided by TS

needed for JSX rendering:

![Screenshot 2024-03-07 at 17 30 11](https://github.com/seamapi/fake-template/assets/7410405/de2deaf2-612b-453c-abd7-4b931d56d575)